src/components/table/table-row/table-row-modern.style.js

```diff
@@ -1,30 +1,79 @@
+import { css } from 'styled-components';
 import StyledTableCell from '../table-cell/table-cell.style';
+import StyledTableHeader from '../table-header/table-header.style';
+import StyledIcon from '../../icon/icon.style';
 
-function applyModernRowStyling({ colors, table }) {
-  return `   
+function applyModernRowStyling(isPassive, { colors, table }) {
+  return css`   
     ${StyledTableCell} {
-      padding-top: 0px;
-      padding-bottom: 0px;
       background-color: ${colors.white};
+
+      ${StyledIcon}::before {
+        transform: rotate(90deg);
+      }
+    }
+
+    ${StyledTableHeader} {
+      padding: 10px 8px;
     }
-    &:not(.carbon-table-row--passive) {
+
+    .custom-drag-layer && {
+      background-color: ${table.dragging};
+      display: block;
+
+      ${StyledTableCell} {
+        background-color: ${table.dragging};
+        border: none;
+      }
+    }
+
+    ${!isPassive && css`
       &:hover {
         ${StyledTableCell} {
           background-color: ${table.primary};
         }
       }
-    }
+    `}
   `;
 }
 
 function applyModernSelectedStyling({ table }) {
   return `
-    background-color: ${table.selected} !important;
-    border-bottom-color: ${table.selected} !important;
+    background-color: ${table.selected};
+    border-bottom-color: ${table.selected};
+  `;
+}
+
+function applyModernDraggedStyling() {
+  return css`
+    border: 1px solid #000A0E; 
```

Comment:
I'm not sure we should to use hardcoded values like this one `#000A0E` inside the dls styles. If we don't have this color  in the color themes maybe it should be good to add this there

Reply:
Good point.

After investigating this more carefully, I discovered that this `border` style isn't actually needed and has no effect. It looks like it was WIP code which @stevebutler2210 was working on. I've now removed it in 8491168.